core/src/test/java/io/questdb/test/std/str/Utf8sTest.java

```diff
@@ -798,15 +798,15 @@ public void testValidateUtf8() {
     @Test
     public void testValidateUtf8Direct() {
```

Comment:
nit:
```suggestion
    public void testGetUtf8SequenceType() {
```

Reply:
done in https://github.com/questdb/questdb/pull/4401/commits/765e27891d2394b5680b41a1483611d61cbb6cc6